ompi/mca/sharedfp/lockedfile/sharedfp_lockedfile_file_open.c

```diff
@@ -125,12 +127,19 @@ int mca_sharedfp_lockedfile_file_open (struct ompi_communicator_t *comm,
             opal_output(0, "[%d]mca_sharedfp_lockedfile_file_open: Error during file open\n", 
                         fh->f_rank);
             free (sh);
-            free(module_data);
+            free (module_data);
             free (lockedfilename);
             return OMPI_ERROR;
         }
-	write ( handle, &position, sizeof(OMPI_MPI_OFFSET_TYPE) );
-	close ( handle );
+	err = opal_best_effort_write ( handle, &position, sizeof(OMPI_MPI_OFFSET_TYPE) );
+        if (OPAL_SUCCESS != err )  {
```

Comment:
There's an extra space between `err` and `)`.

Reply:
done